sycl/doc/extensions/experimental/sycl_ext_oneapi_graph.asciidoc

```diff
@@ -878,6 +1182,20 @@ Exceptions:
 * Throws synchronously with error code `invalid` if `recordingQueue` is
   recording to a different graph.
 
+|
+[source,c++]
+----
+std::vector<node> get_nodes() const;
+----
+|Returns a list of all the nodes present in the graph.
+
+|
+[source,c++]
+----
+std::vector<node> get_root_nodes() const;
+----
+|Returns a list of all nodes in the graph which have no dependencies.
```

Comment:
these two APIs get_*_nodes are added into table "Member functions of the `command_graph` class for queue recording", there should be no node for the queue recording mode, we'd add the two APIs into table "Member functions of the command_graph class" which also contains API "add" etc.